stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -835,6 +838,10 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
     }
   }
 
+  @available(*, unavailable, renamed: "copy(from:bytes:)")
+  public func copyBytes(from source: UnsafeRawPointer, count: Int)
+  { fatalError("unreachable") }
```

Comment:
All function bodies have the format

```swift
public func foo() {
  fatalError()
}
```

See: https://github.com/apple/swift/blob/f6f3ed0fe7e3c0d8d6af85bb238e039041ac0d42/validation-test/stdlib/CollectionDiagnostics.swift#L15